flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/fs/hdfs/HadoopFileSystem.java

```diff
@@ -37,7 +37,7 @@
 public class HadoopFileSystem extends FileSystem {
 
     /** The wrapped Hadoop File System. */
-    private final org.apache.hadoop.fs.FileSystem fs;
+    protected final org.apache.hadoop.fs.FileSystem fs;
```

Comment:
This change is unnecessary. You can use `getHadoopFileSystem()` where it's needed.